### DISCUSSION TITLE
Can’t close iOS app when opened via external external apps

### DISCUSSION THREAD

**Discussion by ByAzuraByAzuraByAzura** [community/community#75260] [Mobile]
The GitHub iOS application repeatedly refuses to close when opened via url links in other apps (attached is a screen recording of this occurring.)

https://github.com/community/community/assets/135807044/cb0cd783-32e6-4aa7-8e38-6f4f5ab5cfa3

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.137.2 (114881244)
Device: iPhone 13 (iOS 17.1.1)
</details>

**Comment by HenSquared** [comment]
Hi @ByAzuraByAzura , 

This is a strange one! At first look, I suspect this is a bug within the OS or within the 3rd party app Launcher. We don't really have control over users opening / closing our app. I was able to reproduce the issue with Launcher and found that if you navigate to another app, like Launcher, closing the app works again. 

I think it might be better to report the bug to Launcher or to Apple.
